### PR TITLE
Update resume-suspend-troubleshooting.md

### PR DESCRIPTION
**Pull request by gw0**

### CODE REVIEW

File: user/troubleshooting/resume-suspend-troubleshooting.md [hunk 1]
```diff
@@ -132,3 +134,8 @@ This can be achieved under a Windows HVM by opening the Device Manager, selectin
 This is illustrated on the screenshot below:
 
 ![r2b1-win7-usb-disable.png](/attachment/wiki/HvmCreate/r2b1-win7-usb-disable.png)
+
+## Suspend turns off the screen and gets stuck
+
+On some devices suspend-to-RAM does not work and a hard power-off is needed to recover, because the system does not go into deep sleep. To get suspend to work, you need to add `mem_sleep_default=deep` to the every `kernel=` line in `/boot/efi/EFI/qubes/xen.cfg` and reboot.
```
Comment (marmarek) [inline user/troubleshooting/resume-suspend-troubleshooting.md:140]: The `/boot/efi/EFI/qubes/xen.cfg` file applies for UEFI boot on R4.0 only. For legacy boot, or UEFI/legacy on R4.1+, the correct place is `/etc/default/grub` (`GRUB_CMDLINE_LINUX` setting).

**Review by marmarek — CHANGES_REQUESTED** [review]

File: user/troubleshooting/resume-suspend-troubleshooting.md (hunk 1 above)
Comment (gw0) [reply user/troubleshooting/resume-suspend-troubleshooting.md]: You are right. I updated the PR accordingly.

**Comment by andrewdavidwong** [comment]
@fepitre @marmarek: Looks like htmlproofer ran out of memory again.

File: user/troubleshooting/resume-suspend-troubleshooting.md [hunk 2]
```diff
@@ -9,6 +9,10 @@ ref: 94
 title: Suspend/Resume Troubleshooting
 ---
 
+# Troubleshooting problems relating to suspend/resume
```
Comment (marmarek) [inline user/troubleshooting/resume-suspend-troubleshooting.md:12]: There is no need to repeat the title (already included in the `title:` above).

**Review by marmarek — CHANGES_REQUESTED** [review]

File: user/troubleshooting/resume-suspend-troubleshooting.md (hunk 2 above)
Comment (gw0) [reply user/troubleshooting/resume-suspend-troubleshooting.md]: Removed and rebased on master.

File: user/troubleshooting/resume-suspend-troubleshooting.md [hunk 3]
```diff
@@ -9,6 +9,8 @@ ref: 94
 title: Suspend/Resume Troubleshooting
 ---
 
+First check if there are any remarks for similar devices in the [Hardware Compatibility List (HCL)](https://www.qubes-os.org/hcl/).
```
Comment (andrewdavidwong) [inline user/troubleshooting/resume-suspend-troubleshooting.md:12]: Please always use relative links for internal website links (see [here](https://www.qubes-os.org/doc/doc-guidelines/#markdown-conventions) for more information).

**Review by andrewdavidwong — CHANGES_REQUESTED** [review]

File: user/troubleshooting/resume-suspend-troubleshooting.md (hunk 3 above)
Comment (gw0) [reply user/troubleshooting/resume-suspend-troubleshooting.md]: Changed to `/hcl/` and according to [conventions](https://www.qubes-os.org/doc/doc-guidelines/#markdown-conventions) rewrapped to max line lenght 80.

**Review by andrewdavidwong — APPROVED** [review]

**Review by marmarek — APPROVED** [review]
```
-----BEGIN PGP SIGNED MESSAGE-----
Hash: SHA256

Approved as of 27951a3f5e4c51e68b9469514232a9ce3514180f
-----BEGIN PGP SIGNATURE-----

iQIzBAEBCAAdFiEEnYWYU8Ll1fIX9iss8yiUvpaEk4oFAmDkRDYACgkQ8yiUvpaE
k4rd0RAApo7ReQKwXS4DOj3p9FgxER5eqVLLzNOK1qSHvpkXqZZaAOK8Ci16pLbj
U33/MgwY8dW6ZbsEP3e7E8rVzG0yfxPhhPPP0bUdHB5NAMXrjrZQrkGsBmr65AU1
JLZuJlgjXo04X8G/aXq6BUnVnnqHm/7ruiIJcNga5URM6tTIaeU8AILYtOuKEgd8
yM4Mr7gdWXpyodb9p1xxfdkwOAYzDxNpL2Zmfdr1WUEnhk2kq2DWEBtXxdiMdTYk
UxjkA0qFepq1rdwrwCmEvTCTCYOziZUGDLC6PXk17E8tbI3yhX9pzzQlFA6iAydF
bGowDa5baLXQRuVIQzZhmsNb3dm2vTvlvcyOhF0WZjthE4ZO0xDsjGSVDIiMTbWK
xYtGrZkmnyxNXX3lq2eLUKj+ZM5DYBIqxx06qCbSbqSW6vt+NMSohj2EhyLV7VO/
VjuqDi2H37UGAL5uKSf8FSi8TIXhMWSelKrHpjRl+mX0fI+M2dyqDXA6akXKvryl
Bycivuc9l3mDP/OFFqi9Ty+YrEj524mBFFE53Dwr5GhaI3SlWuYfWk/jEOIda3oM
LroG6MG+sijQuG9mRINHiantXRFSrYP22znhLTAOTxwDnCd4PinGNErFhmcMPCFo
cDa4V6Vx3NrtiZXyyry6j2HChitsy/8BKVGF7N2CLSYMrvXwL5U=
=v1fp
-----END PGP SIGNATURE-----
```

**Comment by andrewdavidwong** [comment]
@marmarek:
```
$ verify origin/pr/1151
[GNUPG:] GOODSIG F32894BE9684938A Marek Marczykowski-Górecki (Qubes Documentation Signing Key) <marmarek@invisiblethingslab.com>
origin/pr/1151 approved as of 27951a3f5e4c51e68b9469514232a9ce3514180f but currently at 54f2b0e0586de2eb97a859d35419db2750436ece
```

But I only see one commit in this PR: https://github.com/QubesOS/qubes-doc/pull/1151/commits/ad6bcf7fb4d18ef2f7839516c2f6461f0910cdc6

I am confused.

**Review by marmarek — APPROVED** [review]
```
-----BEGIN PGP SIGNED MESSAGE-----
Hash: SHA256

Approved as of ad6bcf7fb4d18ef2f7839516c2f6461f0910cdc6
-----BEGIN PGP SIGNATURE-----

iQIzBAEBCAAdFiEEnYWYU8Ll1fIX9iss8yiUvpaEk4oFAmDkXbkACgkQ8yiUvpaE
k4oUqQ/9F0XHXssIpVRL52yulTqMgdJ9yNDdNN06nuYq12tV0KK9Dbmf8oaPAq1x
ULyCPasgzsWYPgXV8CdKmkRz5qBtIlzGWS9cUNQXE0ULjEf1jt0D8CQttaeo5Tpj
j9OAuaMtMHXK5aG2LYCM9QoJanEZqBvyAnjQrfu2ZhvwVac3mrFBk6LTF1YjcXtp
MBKPbdKwgzJ7T5NgybI0N1aYvh2Rylq9n/CL03PKPxPLCMCe6P6BudClRr/iTQ/Q
7Owq5UAjUW8z0LzSGVYO0akoogP72QjQA6Hx4Cf7k4PY0sayC2v5mh9pBa74yad8
cKfHvCEk9L0CLUrRGoGlTZVlsyuZtfrEK0xLqa7rbLUs2a5S1TrHUToiMmCyoOdm
D0yc23nWU/XAVLTsWjksp/cwIEYSQINfarT/1vPy8ptktYjhG/QbqVXUUlsueJ3b
+whPwk57wzqXKyV4+69s6iAiFcCZEYNRIrIfv1p1ji2Mbldb9Y4GZ9yLV64PVU1o
oCJd7FNtAVBjEJJPeKvx8vpR/Usi1SpQ5ZzyfOquTWFM7sljMjZQz6peWfNbanmy
ldyWpAuxqt50U9WzvGW6kK3FXmbTK9C5k7JE/s8wksReLQJtj8QcoNk+DFMNJW78
ErZOk7T3M7468qAsfahi/iqekC9Y0VgxDcTBBDQF8/8cAr/+sM0=
=Dwmq
-----END PGP SIGNATURE-----
```

**Comment by marmarek** [comment]
See the PR history, I had an earlier version of this commit.

**Pull request merged**